### PR TITLE
Improve readme

### PR DESCRIPTION
**Pull request by ilfa**

### CODE REVIEW

**Review by necipallef — APPROVED** [review]

**Review by necipallef — CHANGES_REQUESTED** [review]
The example code has two incorrect imports, "models.Response" and "models.EventResponse". They're supposed to be "model.Response" and "model.EventResponse"

File: template/README.mustache [hunk 1]
```diff
@@ -247,31 +263,31 @@ To learn more, refer to example located in [src/examples/java/com/fingerprint/ex
 
 ## Webhook signature validation
 This SDK provides utility method for verifying the HMAC signature of the incoming webhook request.
+
+Here is an example how to made it using Spring Boot.
```
Comment (necipallef) [inline template/README.mustache:267]: I'd suggest a different sentence here, such as "Here is an example implementation using Spring Boot"

**Review by necipallef — CHANGES_REQUESTED** [review]
Can we add "import com.fingerprint.sdk.Webhook;" to the Webhook example, please?

File: template/README.mustache (hunk 1 above)
Comment (JuroUhlar) [reply template/README.mustache:267]:
```suggestion
Here is an example implementation using Spring Boot: 
```

Comment (necipallef) [reply template/README.mustache:267]: This suggestion should be applied to `README.md` file as well, given the fact that this file is `.mustache` file.

**Review by necipallef — APPROVED** [review]

**Pull request merged**